pennylane/gradients/pulse_generator_gradient.py

```diff
@@ -619,7 +618,9 @@ def circuit(params):
 
             \omega_{k}^{(\ell)} = \frac{1}{2^N}\mathrm{Tr}\left[P_\ell \Omega_{k}\right]
 
-        where :math:`N` is the number of qubits.
+        where :math:`N` is the number of qubits and :math:`\ell = 1, .. , L` the Pauli word index.
+        The number of non-zero Pauli words :math:`L` is the dimension of the dynamical Lie algebra
+        and at most :math:`4^N-1`.
```

Comment:
I guess this _can_ be incorrect, if we happen to compute the effective generator at a weird position in parameter space, right? :thinking: Maybe inserting a "typically" or "usually" or so could defuse this?

Reply:
Ah you mean even though the dynamical lie algebra has some size, it could be that some coefficients in the decomposition are zero?

Reply:
updated the sentence